shared-test/string-utils.ts

```diff
@@ -9,3 +9,7 @@ export const randomString = (length: number): string => {
   }
   return result;
 };
+
+export const renderDid = (didController: string, keyId: string): string => {
```

Comment:
Personally I think this is better in `shared-test/jwt-utils.ts` as this `string-utils` is for domain non-specific things in my head.